apps/asn1pars.c

```diff
@@ -258,14 +258,14 @@ int asn1parse_main(int argc, char **argv)
         num = tmplen;
     }
 
-    if (offset >= num) {
+    if (offset < 0 || offset >= num) {
         BIO_printf(bio_err, "Error: offset too large\n");
```

Comment:
Now `"Error: offset too large\n"` sounds a bit funny for negative arguments. Maybe you could change this to `"Error: offset out of range\n"`, if this does not conflict with cherry-picking?

Reply:
No, I guess that will be okay.
I will do that in an extra commit, so it won't break anything.

Reply:
And although said already implicitely between the lines: cherry-pick approved for 1.0.2.